libc/src/__support/FPUtil/cast.h

```diff
@@ -26,9 +26,18 @@ LIBC_INLINE constexpr cpp::enable_if_t<cpp::is_floating_point_v<OutType> &&
                                            cpp::is_floating_point_v<InType>,
                                        OutType>
 cast(InType x) {
+  //  Casting to the same type is a no-op.
```

Comment:
Really tiny nit: now there are 2 spaces after the `//`.

```suggestion
  // Casting to the same type is a no-op.
```